capstone-app/src/app/modules/history/wiki-results.service.ts

```diff
@@ -1,8 +1,19 @@
 import { Injectable } from '@angular/core';
+import { WikiSearchResult } from './WikiSearchTemplate';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
+
 export class WikiResultsService {
-  constructor() { }
+  constructor(private readonly http: HttpClient) {}
+  private static readonly URL_BEGINNING = 'https://en.wikipedia.org/w/api.php?origin=*&action=parse&page=';
+  private static readonly URL_END = '&format=json';
```

Comment:
If I recall correctly, capitalizing all of url is against google style guide. I’m not sure if this rule has an exception to static readonly variables like here. Matt, Nicholas?

Reply:
I thought static, read only are supposed to be capitalized...